lib/watir/elements/table_cell.rb

```diff
@@ -2,5 +2,12 @@ module Watir
   class TableCell < HTMLElement
     alias_method :colspan, :col_span
     alias_method :rowspan, :row_span
+
+    def column_header
+      index = self.previous_siblings.size
+      table = self.parent(tag_name: 'table')
+      TableHeaderCell.new(table.tr, {tag_name: 'th', index: index})
```

Comment:
I've never #hashes, but it looks like it makes a ridiculous number of wire calls, most of which won't be needed to handle the basic use cases.

I'll take a look, though on how it is structured differently. Thanks!